lib/rollbar/rails.rb

```diff
@@ -1,5 +1,15 @@
 module Rollbar
   module Rails
 
+    def self.included(base)
+      base.extend ClassMethods
+    end
+
+    module ClassMethods
+      def on_exception_log_values(*vars)
+        Rollbar.configuration.custom_values.merge!({ self.name.parameterize => [*vars].map(&:to_s) })
```

Comment:
I'd prefer store this data in the controller itself. Another controller method can return the custom data for it according to stored `vars`.